### ISSUE TITLE
pyportify-copyall issue when Creating in Google Music

### ISSUE THREAD

**Issue by luislobo** [rckclmbr/pyportify#97]
```
Creating in Google Music... Traceback (most recent call last):
  File "/usr/local/bin/pyportify-copyall", line 9, in <module>
    load_entry_point('pyportify==0.3.13', 'console_scripts', 'pyportify-copyall')()
  File "/usr/local/lib/python3.5/dist-packages/pyportify/copy_all.py", line 62, in main
    loop.run_until_complete(start())
  File "/usr/lib/python3.5/asyncio/base_events.py", line 387, in run_until_complete
    return future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/usr/lib/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "/usr/local/lib/python3.5/dist-packages/pyportify/copy_all.py", line 56, in start
    yield from app.transfer_playlists(None, s, g, playlists)
  File "/usr/local/lib/python3.5/dist-packages/pyportify/app.py", line 177, in transfer_playlists
    yield from g.add_songs_to_playlist(playlist_id, gm_track_ids)
  File "/usr/local/lib/python3.5/dist-packages/pyportify/google.py", line 95, in add_songs_to_playlist
    added_ids = [e['id'] for e in res['mutate_response']]
  File "/usr/local/lib/python3.5/dist-packages/pyportify/google.py", line 95, in <listcomp>
    added_ids = [e['id'] for e in res['mutate_response']]
KeyError: 'id'
```

Is there anything else you might need for me to test?

**Comment by luislobo** [comment]
It seems to happen when you want to transfer a playlist that has more than 1000 songs (that's google play limit) May be you can split the list into two, adding playlistname-2 playlistname-3 playlistname-4 for every thousand?

**Comment by luislobo** [comment]
https://github.com/rckclmbr/pyportify/pull/93

**Comment by luislobo** [comment]
Yeah, I actually just verified that that PR fixes my issue

**Comment by rckclmbr** [comment]
This should be fixed in v0.3.14, give it a try and let me know.
